common/src/main/java/com/wynntils/commands/ConfigCommand.java

```diff
@@ -607,6 +725,24 @@ private Overlay getOverlayFromArguments(
         return overlayOptional.get();
     }
 
+    private static OverlayGroupHolder getOverlayGroupHolderFromArguments(
+            CommandContext<CommandSourceStack> context, String featureName, String overlayGroupName) {
+        Feature feature = getFeatureFromArguments(context, featureName);
+        if (feature == null) return null;
```

Comment:
Ok, my bad, the error message should go here, not where you called this function